src/ovp_pipeline/promote_candidates.py

```
@@ -378,31 +392,49 @@ def list_candidates(registry: ConceptRegistry) -> None:
         print()
 
 
-def review_candidates(registry: ConceptRegistry) -> list[tuple[ConceptEntry, str, list]]:
+def review_candidates(
+    registry: ConceptRegistry,
+    *,
+    pack: "BaseDomainPack | None" = None,
```

Ruff (0.15.10)

[error] Undefined name `BaseDomainPack`

(F821)